scripts/eurostat/health_determinants/common/denominator_mcf_generator.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import re
+import os
+import sys
+
+_COMMON_PATH = os.path.abspath(os.path.dirname(__file__))
+sys.path.insert(1, _COMMON_PATH)
+from dcid_existance import check_dcid_existance
+
+
+def _generate_mcf(sv_list, mcf_file_path) -> None:
+    """
+    This method generates MCF file w.r.t
+    dataframe headers and defined MCF template
+
+    Args:
+        sv_list (list) : List of DataFrame Columns
+
+    Returns:
+        None
+    """
+    # pylint: disable=R0914
+    # pylint: disable=R0912
+    # pylint: disable=R0915
+    actual_mcf_template = ("Node: dcid:{}\n"
+                           "name: \"{}\"\n"
+                           "typeOf: dcs:StatisticalVariable\n"
+                           "populationType: dcs:Person{}{}{}{}{}{}{}{}{}{}{}\n"
```

Comment:
Instead of so many {}, you can do something like this: https://github.com/datacommonsorg/data/blob/master/scripts/us_census/pep/pep_by_asrh/preprocess.py#L1102